### PR TITLE
Additional fix: Refreshing stopped if no change in location

### PR DESCRIPTION
**Pull request by 97balakrishnan**
## Description

Additional Fix to #264

- When no change to location occurs the refresh of nearby location doesn't occur now.
- This change has been incorporated for both Pull down gesture refreshing and for the Refresh Button
- I added a flag to ensure that OnLocationChanged() doesn't call refreshView() when location hasn't changed.

### CODE REVIEW

**Comment by 97balakrishnan** [comment]
Oh ok.I'll do that.

**Comment by 97balakrishnan** [comment]
Done.

File: app/src/main/java/fr/free/nrw/commons/nearby/NearbyActivity.java [hunk 1]
```diff
@@ -46,7 +46,7 @@
 
     private static final int LOCATION_REQUEST = 1;
     private static final String MAP_LAST_USED_PREFERENCE = "mapLastUsed";
-
+    private boolean LOCATION_CHANGED=false;
```
Comment (maskaravivek) [inline app/src/main/java/fr/free/nrw/commons/nearby/NearbyActivity.java]: please refer to google style guide for private variable naming conventions. :)

Comment (97balakrishnan) [reply app/src/main/java/fr/free/nrw/commons/nearby/NearbyActivity.java]: Sorry :p . I know that its an elementary mistake, I'll correct it.

**Comment by neslihanturan** [comment]
Thanks for the PR. However there will be some critical changes with nearby activity. So I suggest wait and see if we still need this after them.

**Comment by 97balakrishnan** [comment]
Ok thanks

**Comment by neslihanturan** [comment]
Closing currently, we can olways re-open it to see if it is still required after nearby activity changed.

**Comment by 97balakrishnan** [comment]
ok :+1:

**Pull request closed without merging**